lib/sync.py

```diff
@@ -164,7 +165,7 @@ def synchronize_account(self, syncpair):
         log.info("Retrieved {} ynab transactions...".format(len(transactions)))
 
         # Push start date back to latest YNAB entry
-        if not get_all:
+        if not replay_bunq_tx:
             start_dt = min(start_dt, get_last_transaction_date(transactions))
```

Comment:
Retrieving bunq transactions is quite slow. The "push back" is there to avoid retrieving too many bunq transactions.  For older accounts like mine, that causes a timeout in Amazon.

What do you think of pushing back a number of days instead of fully?

Reply:
That's a fair point. Do you think that transforming the parameter from a flag to a date or number of days would go the right way?

Reply:
Overwriting uncleared transactions seems fine to me me.

What's the added value of the `--replay-bunq-tx` flag? What does it do besides setting the `--all` flag?

What do you think of using the date of the last uncleared transaction of the past 30 days as start time for retrieving bunq transactions?

Reply:
> What do you think of using the date of the last uncleared transaction of the past 30 days as start time for retrieving bunq transactions?

That seems like a cleaner way to do this, will piggy back on the "35 days back".